docs/csharp/language-reference/keywords/toc.md

```diff
@@ -86,6 +86,7 @@
 ### [lock Statement](lock-statement.md)
 ## [Method Parameters](method-parameters.md)
 ### [params](params.md)
+### [in](in.md)
 ### [ref](ref.md)
 ### [out](out.md)
```

Comment:
unrelated to your PR: this should be removed and out-parameter-modifier.md promoted (unless you consider a generic type parameter in a method signature a method parameter, in which case out-generic-modifier.md should be added.

Reply:
I added the other uses for out and in to the TOC.